### ISSUE TITLE
Unable to deal with captcha.

### ISSUE THREAD

**Issue by leovan** [leovan/SciHubEVA#5]
It seems that we need enter captcha sometimes to download the PDF from sci-hub.

**Comment by ziofat** [comment]
![image](https://user-images.githubusercontent.com/8521174/43587145-158b9566-969c-11e8-8dac-a88a37058b88.png)

Captcha image did not show up.

**Comment by leovan** [comment]
Thanks for report. I tried with this case, actually the response of fetching the pdf is not the page with captcha. Since the response is an HTML and it also has an <iframe>, the <iframe> is also an HTML (which was used to determine whether we need give the captcha or not). But the truth is that in the <iframe> is an HMTL, yet not the CAPTCHA page. I tried with sci-hub.tw too, it leads me to another page to download it (https://sci-hub.tw/https://epjdatascience.springeropen.com/articles/10.1140/epjds/s13688-018-0146-8) and it seems an OPEN ACCESS articles. So I think I can't handle all scenarios of different external websites. I think I can enhance the log output, which may help user to download manually. And I will fix with this later in next release.
